### PR TITLE
[Improve] add database file server provider

### PR DESCRIPTION
**Pull request by pwallk**
## What's changed?

- add local database file server provider
- improve define yml delete

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+@NoArgsConstructor
+@Schema(description = "monitor define entity")
+@EntityListeners(AuditingEntityListener.class)
+public class Define {
+
+    @Id
+    @Schema(title = "app", example = "websocket")
+    private String app;
+
+    @Column(columnDefinition = "text")
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java:54]: hi,  `columnDefinition = "text"` works with MySQL and PostgreSQL, but less portable. suggest use `@Lob`

File: common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+@Schema(description = "monitor define entity")
+@EntityListeners(AuditingEntityListener.class)
+public class Define {
+
+    @Id
+    @Schema(title = "app", example = "websocket")
+    private String app;
+
+    @Lob
+    @Column(columnDefinition = "text")
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java:56]: @Lob and `@Column(columnDefinition = "text")` are duplicates

File: common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java (hunk 1 above)
Comment (pwallk) [reply common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java:56]: I'm going to remove that

File: common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java (hunk 2 above)
Comment (pwallk) [reply common/src/main/java/org/apache/hertzbeat/common/entity/manager/Define.java:56]: I'm going to remove that

**Review by tomsun28 — APPROVED** [review]
lgtm

**Pull request merged**